### DISCUSSION TITLE
Osu!Lazer Hitsounds not functioning on Sliders in editor after transferring from stable (2022.108.0)

### DISCUSSION THREAD

**Discussion by Imprixie** [ppy/osu#16376] [Q&A]
I've recently switched to the new osu!lazer build. I've transferred all my maps, and one of them was unfinished. I started working on it. I hitsound while I map, and very quickly realized that when I played the map after adding hitsounds, sliders never made sound. I doublechecked to see if the hitsound, in this case clap, was enabled on the slider. It was, but no sound played. I was recommended to report this issue here. Here's what I found:

- Sliders that were originally in the map before transferring from stable do not play the selected hitsounds. I tested for whistle and finish as well; neither work.
- Copy-pasting a pre-transfer slider with a hitsound attached, or copy-pasting a slider and _then_ adding a hitsound does not fix the issue.
- Creating a new slider with a hitsound selected, or adding a hitsound to a newly made slider _does_ work.
- Creating a new slider, exiting the editor, reentering it, and adding a hitsound does not work. 
- Creating a new slider, adding a hitsound, exiting the editor, and reentering it causes the head of the slider to not play a hitsound, but the end of the slider still works(?!).

- This issue persists with other maps, whether my garbage circle spam, or a ranked map. Removing hitsounds from ranked maps also seemed unsuccessful; the hitsound plays anyways.  

- This issue persists after restarting osu!.

Any instance where I stated the hitsound does and does not work applies both in the **editor** and in **gameplay**.
I did not move any sliders (including new ones) or touch any other object in the editor. I am not familiar with programming in the slightest, except for a little html, so I don't trust my own "expertise" in trying to deduce the issue. 

I could not find any other issues similar enough to this one. If this issue has already been reported I apologize.

I am using the latest version (2022.108.0 as of writing this) and updated just before this issue happened. Running on Microsoft Windows 10.

[updater.log](https://github.com/ppy/osu/files/7834503/updater.log)
[database.log](https://github.com/ppy/osu/files/7834504/database.log)
[network.log](https://github.com/ppy/osu/files/7834505/network.log)
[performance.log](https://github.com/ppy/osu/files/7834506/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7834507/runtime.log)

**Comment by bdach** [comment]
can you attach the map and a video of this?

sounds like this may be #15618 but hard to say anything for sure. just importing the map shouldn't have nuked samples.

**Reply by Imprixie** [reply]
yes, of course. I'm a little slow in the video, still new to the lazer build and don't know my hotkeys by heart. If there's something specific you'd like me to record I can do it, as I am not exactly proficient at bug reporting.

I have seen #15618 but because I haven't moved any objects, I just wanted to make sure. 

I'm putting in the map file, but I was able to replicate the issue with seemingly random ranked maps.

both the osu! file and MKV file are unsupported so I'm making it a google drive folder.
https://drive.google.com/drive/folders/1mEyWYirOcLBXvVJUIRiArqHx-LsvYoLT?usp=sharing

**Reply by peppy** [reply]
Can you please record videos using the default skin? I have no idea what I'm listening to. Not really helpful sorry.

**Reply by Imprixie** [reply]
I don't understand what's unclear about the video? I could understand if it was a standardization thing but this response is unreasonable at best. Regardless, it's been a month and I don't have access to osu!lazer. I don't have space to reinstall on my computer as of now, and I don't know when I'll have enough space cleared up. I assumed the discussion was lost to the void as well.

Regardless, there is a **very** loud clap hitsound that reappears multiple times and contrasts harshly with the default. The hitsounds I'm referring to would be _that._ Also, the music volume is at 7% if I recall correctly. Please try to listen again, I'm sure you can do it if you try hard enough.

Or perhaps you hadn't had the time to go past _55 seconds_ into the video to hear them. Either way, I am unable to help as of now.

Best of luck.

**Reply by peppy** [reply]
I did not watch the whole video, no. If you can provide timestamps of where the issue is, that may be a good start.
